lib/ansible/galaxy/role.py

```diff
@@ -242,6 +242,11 @@ def install(self):
                                                                                                                                          self.name,
                                                                                                                                          role_versions))
 
+                # check if there's a source link for our role_version
+                for role_version in role_versions:
+                    if role_version['name'] == self.version and 'source' in role_version:
+                        self.src = role_version['source']
```

Comment:
We can name this key whatever. I picked 'source' because it corresponded to src and url was already taken in the galaxy api.